src/Augmentation/AugmentationHelpers.jsx

```diff
@@ -1611,8 +1609,8 @@ function initAugmentations() {
              "The CRTX42-AA is an artificially-synthesized gene that targets the visual and prefrontal " +
              "cortex and improves cognitive abilities.<br><br>" +
              "This augmentation:<br>" +
-             "Improves the player's hacking skill by 8%.<br>" +
-             "Improves the player's hacking experience gain rate by 15%.",
+             "Improves hacking skill by 8%.<br>" +
```

Comment:
Lets change "Improves" to "Increases" to be consistent with others

Reply:
Done